### ISSUE TITLE
comparison with 0 cause error

### ISSUE THREAD

**Issue by 1vanK** [kasparsklavins/bigint#5]
```
            Bigint bi = 0;
            if (bi == 0)    // crash
                bi = 1;
```

**Comment by kasparsklavins** [comment]
There's an infinite loop in the comparison method, might be related to that https://github.com/kasparsklavins/bigint/blob/master/src/bigint.cpp#L256
I'll look into it.

**Comment by 1vanK** [comment]
bigint = 0  ->  number.size() == 0   ->  number.size() - 1 == max_unsigned --> out of range

**Comment by kasparsklavins** [comment]
Solved by #7
